README.md

```diff
@@ -181,11 +181,10 @@ You can deploy this project using:
 
   >
 
-👤 **Author2**
+👤 **Haben Mehari**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@habitoz](https://github.com/habitoz)
+- Linkedin: [@habitoz](https://www.linkedin.com/in/haben-mehari-593531140)
```

Comment:
- [ ] Your project functions as expected, You just need to make some changes to your Readme file before moving to the next milestone. :+1: :100: 

- - As having a professional-looking readme is important, I think there are things to update here.
- - - Your readme file still has some placeholder texts, it is required that you remove all the guiding comments while customizing your readme. See the screenshot below. Also, you have a comment in line `112 and 225`
- - - You are also missing a project title, I'll suggest you update that too.
- - Screenshot:
![Selection_054](https://github.com/Yonashenok/Awesome-books/assets/107632780/1adcbf17-c31d-4306-8149-139a4931a06d)
 
- - - Your MIT license link leads to a 404 error page, A broken link in the reame makes it look a bit unprofessional, I'll suggest you update that also.
- - SCREENSHOT:
![Screenshot from 2023-06-05 10-20-05](https://github.com/Yonashenok/Awesome-books/assets/107632780/a7d88fe0-b3d7-4017-9667-e63afaf83d35)